### DISCUSSION TITLE
SSH Tunnel Jump Server

### DISCUSSION THREAD

**Discussion by isonnymichael** [dbeaver/dbeaver#34664] [Q&A] [answered]
I want to connect to the database on server B but through server A, as there are some reasons why I cannot directly access server B.

I can access server B by SSHing into server A and then SSHing with a config file to server B (without a password).

I've tried using Jump servers feature, but I still can't connect SSH to server B. I need to SSH connect to server B using the private key that exists on server A, because I don't have access the password on server B, how can I do that?

![346003475-a5e77645-e371-4204-be4c-797ddb769252](https://github.com/user-attachments/assets/4d389824-2c45-4d47-ab3c-b91835b9cfc1)

**Comment by E1izabeth** [comment, marked as answer]
Currently it's not possible. However, you can create feature request

**Reply by isonnymichael** [reply]
Thank you, I have already made the feature request. I hope this feature can be created soon.

https://github.com/dbeaver/dbeaver/issues/34816
